intelmq/bots/outputs/restapi/output.py

```diff
@@ -34,7 +34,13 @@ def process(self):
                               cert=self.ssl_client_cert,
                               timeout=self.http_timeout_sec,
                               **kwargs)
-        r.raise_for_status()
+        if self.parameters.logging_level in ["DEBUG", 10] and r.status_code != 200:
+            try:
+                self.logger.debug("Something failed during message sending with response body {}".format(r.text))
+            except Exception as e:
+                self.logger.debug("There was an error in your message, but there was no response text {}".format(e))
```

Comment:
Isn't that covered by `r.raise_for_status` already?

Reply:
I've added this block as a precaution in case there is no text in response just to be sure. I haven't managed to find any info whether some text will always be present or not.

Reply:
> I've added this block as a precaution in case there is no text in response just to be sure.

Sure, but isn't that what's `raise_for_status` does? And the method also includes more information (error code and message).

> I haven't managed to find any info whether some text will always be present or not.

Is `r.text` then `None` or does it not exist at all ( = AttributeError)?